src/types/PostTypes.tsx

```diff
@@ -0,0 +1,26 @@
+type Comment = {
+  username: string;
+  commentText: string;
+};
+
+// type Comment = Pick<User, "username"> & { commentText: string };
+
+export type User = {
+  username: string;
+  profilePic: string;
+  storyImages?: string[];
+  name?: string;
+};
+
+export type CurrentUser = Pick<User, "username" | "profilePic" | "name">;
+export type SuggestUser = User;
```

Comment:
Maybe just use the same type, no need to introduce "SuggestUser"

Consider removing "CurrentUser" too, and just leave some fields unused, it's fine.